src/app/src/main/kotlin/org/imperial/mrc/hint/APIClient.kt

```diff
@@ -6,17 +6,18 @@ import org.springframework.http.ResponseEntity
 import org.springframework.stereotype.Component
 
 interface APIClient {
-    fun validate(path: String, type: FileType): ResponseEntity<String>
+    fun validateBaseline(path: String, type: FileType): ResponseEntity<String>
```

Comment:
Support for validation all baseline data together will come later? As a separate function called something like `validateBaselineAll`? I can update the endpoints on the R side from `validate/input` and `validate/baseline` to `validate/baseline` and `validate/baseline/all` so we are agreeing on our names.

Reply:
Yep, that all sound good to me